scapy/packet.py

```diff
@@ -100,7 +112,7 @@ class Packet(six.with_metaclass(Packet_metaclass,  # type: ignore
         "wirelen",
     ]
     name = None
-    fields_desc = []  # type: List[Field[Any, Any]]
+    fields_desc = []  # type: Sequence[AnyField]
```

Comment:
Do you still need to use Sequence?

Reply:
I think so. Without it I get an error:

        scapy/layers/l2.py:620: error: Incompatible types in assignment (expression has type "List[IntEnumField]", base class "Packet" defined the type as "List[Union[Field[Any, Any], _FieldContainer]]")
        scapy/layers/l2.py:620: note: "List" is invariant -- see http://mypy.readthedocs.io/en/latest/common_issues.html#variance
        scapy/layers/l2.py:620: note: Consider using "Sequence" instead, which is covariant

Reply:
Fair enough !